go/controller/internal/autogen/autogen_api_translator.go

```diff
@@ -27,7 +27,8 @@ var (
 		"kagent.tools.k8s.GenerateResourceTool",
 	}
 	toolsProvidersRequiringOpenaiApiKey = []string{
-		"kagent.tools.docs.QueryTool",
+		"query_docs",
+		"kagent-querydoc",
 	}
```

Comment:
Can we remove these maps as we decided the config no longer lives at this level anyway? Now the config lives in the container itself iirc.